dbschema/funding-account.esdl

```diff
@@ -0,0 +1,16 @@
+module default {
+  type FundingAccount extending Resource, Mixin::Named {
+    overloaded name {
+      constraint exclusive;
+    }
+    
+    required accountNumber: FundingAccount::accountNumber;
+  }
+}
+
+module FundingAccount {
+  scalar type accountNumber {
```

Comment:
Idk how necessary a separate type is. In other places we kinda justified the guess by saying it could be used in more places. `population` could be used in multiple spots, etc.
It seems really unlikely that this will ever be used anywhere else. We could just inline the constraints.
Thoughts?

Reply:
I'm totally fine with that.